Doc/library/doctest.rst

```diff
@@ -584,20 +583,7 @@ doctest decides whether actual output matches an example's expected output:
    Note that :const:`ELLIPSIS` can also be used to ignore the
    details of the exception message, but such a test may still fail based
    on whether or not the module details are printed as part of the
```

Comment:
It was like this before your PR, but I don't really understand what this sentence means.

Reply:
Hmm, maybe—but it will still fail if the name or the exception module is changed, it is moved to a different one or it is just not specified in the doctest, which is still relevant regardless of Python version, no? Perhaps this should just be reworded to something like 

```suggestion
   Note that :const:`ELLIPSIS` can also be used to ignore the
   details of the exception message, but such a test may still fail based
   on whether the module is specified or matches exactly in the
```

But if that isn't an important-enough use case or one we want to encourage in modern Python, perhaps it justifies no longer documenting it?